### PR TITLE
Update some dependencies not detected by dependabot

### PR DESCRIPTION
**Pull request by koushiro**
Signed-off-by: koushiro <koushiro.cqx@gmail.com>

### CODE REVIEW

**Comment by koushiro** [comment]
@mxinden PTAL
It seems to be a bug of `dependabot`

File: protocols/secio/Cargo.toml [hunk 1]
```diff
@@ -13,9 +13,9 @@ categories = ["network-programming", "asynchronous"]
 maintenance = { status = "deprecated" }
 
 [dependencies]
-aes-ctr = "0.3"
-aesni = { version = "0.6", features = ["nocheck"], optional = true }
-ctr = "0.3"
+aes-ctr = "0.5"
```
Comment (mxinden) [inline protocols/secio/Cargo.toml:16]: Secio is deprecated, see https://github.com/libp2p/rust-libp2p/pull/1764.

File: protocols/secio/Cargo.toml [hunk 2]
```diff
@@ -13,9 +13,9 @@ categories = ["network-programming", "asynchronous"]
...
+aesni = { version = "0.9", features = ["nocheck"], optional = true }
```
Comment (mxinden) [inline protocols/secio/Cargo.toml:17]: Secio is deprecated, see https://github.com/libp2p/rust-libp2p/pull/1748.

File: protocols/secio/Cargo.toml [hunk 3]
```diff
@@ -13,9 +13,9 @@ categories = ["network-programming", "asynchronous"]
...
+ctr = "0.5"
```
Comment (mxinden) [inline protocols/secio/Cargo.toml:18]: Secio is deprecated, see https://github.com/libp2p/rust-libp2p/pull/1764.

File: protocols/secio/Cargo.toml [hunk 4]
```diff
@@ -28,7 +28,7 @@ rand = "0.7"
 rw-stream-sink = "0.2.0"
 sha2 = "0.9.1"
 static_assertions = "1"
-twofish = "0.2.0"
+twofish = "0.4.0"
```
Comment (mxinden) [inline protocols/secio/Cargo.toml:31]: Secio is deprecated, see https://github.com/libp2p/rust-libp2p/pull/1756.

File: Cargo.toml [hunk 5]
```diff
@@ -80,7 +80,7 @@ libp2p-wasm-ext = { version = "0.22.0", path = "transports/wasm-ext", optional =
 libp2p-yamux = { version = "0.25.0", path = "muxers/yamux", optional = true }
 multiaddr = { package = "parity-multiaddr", version = "0.9.3", path = "misc/multiaddr" }
 multihash = "0.11.0"
-parking_lot = "0.10.0"
+parking_lot = "0.11.0"
```
Comment (mxinden) [inline Cargo.toml:83]: It is indeed surprising that dependabot didn't catch this and the ones below. I am guessing this is due to some crates already using `parking_lot` `v0.11`, e.g. [libp2p-yamux](https://github.com/libp2p/rust-libp2p/blob/master/muxers/yamux/Cargo.toml#L15).

Let's align all crates to the same `parking_lot` version with this pull request :+1:.

**Review by mxinden — COMMENTED** [review]
Thanks for this pull request.

Would you mind reverting the secio related changes and adding the relevant changelog and cargo toml changes for the `pnet` crate similar to https://github.com/libp2p/rust-libp2p/pull/1772/commits/eb8ec997ee56253c591cdede9523f69b3021e574 ?

**Review by mxinden — APPROVED** [review]
Thanks for the quick follow-up. This looks good to me.

Giving others a chance to review as well. Will merge later today or tomorrow in case there are no objections.

**Review by romanb — APPROVED** [review]

**Comment by mxinden** [comment]
Ah, missed one last thing:

```
libp2p-pnet = { version = "0.19.1", path = "protocols/pnet", optional = true }
```

should be

```
libp2p-pnet = { version = "0.19.2", path = "protocols/pnet", optional = true }
```

in the root `Cargo.toml`.

**Comment by koushiro** [comment]
Oops, I forgot about that. It's fixed now. @mxinden

**Comment by mxinden** [comment]
Thanks @koushiro.

**Pull request merged**